tiingo/src/index.ts

```diff
@@ -0,0 +1,7 @@
+import { expose, util } from '@chainlink/ea-bootstrap'
+import { makeExecute } from './adapter'
+import { makeConfig } from './config'
+
+const NAME = 'TIINGO'
+
+export = { NAME, makeExecute, makeConfig, ...expose(util.wrapExecute(makeExecute())) }
```

Comment:
A very recent change got rid of the need for `wrapExecute`.

```suggestion
export = { NAME, makeExecute, makeConfig, ...expose(makeExecute()) }
```